src/main/java/frc/robot/subsystems/tower/ArmImpl.java

```diff
@@ -53,7 +84,7 @@ public Position getPosition() {
 
     @Override
     public void setSpeed(double speed) {
-        double maxPower= 0.25;
+        double maxPower = 0.25;
```

Comment:
We shouldn't be limiting motor output anymore once a PID loop is running. Or if in fact we do need to cap the power output then there are better ways to do this.

Reply:
That was there before I did PID, it's probably unneeded now though like you said

Reply:
I know it was there from before, since it was there for safety during manual testing. With added PID however it will interfere with tuning.

Reply:
Mirroring what was just said on Slack, this power limit will actually be circumvented by the onboard PID controller since that won't call through to setSpeed. For safety reasons Steve recommends we retain a power limiter, but this should be done by calling the relevant configuration method(s) on the Talon object in the ArmImpl constructor.

Reply:
https://github.com/CrossTheRoadElec/Phoenix-Examples-Languages/blob/master/Java/PositionClosedLoop_AuxFeedForward/src/main/java/frc/robot/Robot.java#L145